src/Rules/Library/IsContentElementTrueOptional.cs

```diff
@@ -20,11 +20,12 @@ public IsContentElementTrueOptional()
             this.Info.HowToFix = HowToFix.IsContentElementTrueOptional;
             this.Info.Standard = A11yCriteriaId.ObjectInformation;
             this.Info.PropertyID = PropertyType.UIA_IsContentElementPropertyId;
+            this.Info.ErrorCode = EvaluationCode.Open;
         }
 
-        public override EvaluationCode Evaluate(IA11yElement e)
+        public override bool PassesTest(IA11yElement e)
         {
-            return EvaluationCode.Open;
+            return true;
```

Comment:
Won't this result in an evaluation code of Pass instead of Open?

Reply:
I suspected I might make that mistake at some point. Nice catch.